db/migrate/20240730090211_remove_proxy_byos_boolean.rb

```diff
@@ -0,0 +1,11 @@
+class RemoveProxyByosBoolean < ActiveRecord::Migration[6.1]
+  def up
+    safety_assured { remove_column :systems, :proxy_byos }
+  end
+
+  def down
+    add_column :systems, :proxy_byos, :boolean, default: false
+    System.where(proxy_byos_mode: 0).in_batches.update_all proxy_byos: false
```

Comment:
This is not necessary, since `default: false`.

Reply:
good catch, removed the unnecessary from the up method, missed the down one